lib/rubocop/cop/rspec/mixin/skip_or_pending.rb

```diff
@@ -10,13 +10,33 @@ module SkipOrPending
         # @!method skipped_in_metadata?(node)
         def_node_matcher :skipped_in_metadata?, <<-PATTERN
           {
-            (send _ _ <#skip_or_pending? ...>)
-            (send _ _ ... (hash <(pair #skip_or_pending? { true str dstr }) ...>))
+            (send _ _ <#skip_or_pending_sym? ...>)
+            (send _ _ ... (hash <(pair #skip_or_pending_sym? { true str dstr }) ...>))
           }
         PATTERN
 
-        # @!method skip_or_pending?(node)
-        def_node_matcher :skip_or_pending?, '{(sym :skip) (sym :pending)}'
+        # @!method skip_or_pending_sym?(node)
+        def_node_matcher :skip_or_pending_sym?, '{(sym :skip) (sym :pending)}'
+
+        # @!method skip_or_pending_inside_block?(node)
+        #   Match examples defined inside a block which is not a hook
```

Comment:
> examples defined inside a block

> which is not a hook

Are those correct?
It appears to me that this matcher is more like "Match skip/pending statements inside a block".